src/main/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClient.java

```diff
@@ -83,50 +83,22 @@ public Health serviceHealthy() {
     }
 
     private URI htmlEndpoint() {
-        return pdfServiceBaseUrl.resolve(
-            String.format("/api/%s/pdf-generator%s", version, GENERATE_FROM_HTML_ENDPOINT_PATH)
-        );
+        return pdfServiceBaseUrl.resolve("/pdfs");
```

Comment:
this could be now set in a constructor, no need to call a function every time.

Reply:
To keep things consistent `/health` endpoint should be handled in the same manner. Refactoring...